File: C/Iniciantes/Basicos/Uri1009_SalarioComBonus.c
#include<stdio.h>

int main(){
	
	double sal, vendas, total;
	char nome[10];
	
	scanf("%s", &nome);
	scanf("%lf %lf", &sal, &vendas);
	
	total = sal + (0.15 * vendas);
	
	printf("TOTAL = R$ %.2lf\n", total);
		
	return 0;
}

File: Javascript/uri1009_SalarioComBonus.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split("\n");

var nome = lines.shift();
var salFixo = parseFloat(lines.shift());
var vendas = parseFloat(lines.shift());

function salarioComBonus(salFixo, vendas){
    return ((vendas * (15/100)) + salFixo);
}

console.log("TOTAL = R$ " + salarioComBonus(salFixo, vendas).toFixed(2));